test/e2e/frictionless/features/regression/converter.feature

```diff
@@ -82,4 +82,39 @@ Feature: Frictionless Converter Block
 
   Examples:
       | Verb     | File                |
-      | sign-pdf | test-files/test.pdf |
\ No newline at end of file
+      | sign-pdf | test-files/test.pdf |
+
+  @MWPW-135907 @regression @converter
+  Scenario Outline: L2 Verbs - Upload, rotate and download
+    Given I go to the <Verb> page
+    Then I upload the files "<Files>"
+    Then I rotate right the uploaded files
+    Then I rotate left first uploaded file
+    Then I wait for 2 seconds
+    Then I save rotated files
+    Then I download the converted file
+
+    Examples:
+      | Verb       | Files                                    |
+      | rotate-pdf | test-files/test.pdf,test-files/test2.pdf |
+
+  @MWPW-135907 @regression @converter
```

Comment:
Please log a Test ticket and use the Jira ID as tag here.

Reply:
This is also resolved, please check